### PR TITLE
Add alt tags to images

### PR DESCRIPTION
**Pull request by mal-tee**
> Omitting alt altogether indicates that the image is a key part of the content and no textual equivalent is available. Setting this attribute to an empty string (alt="") indicates that this image is not a key part of the content (it’s decoration or a tracking pixel), and that non-visual browsers may omit it from rendering. Visual browsers will also hide the broken image icon if the alt is empty and the image failed to display."

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

### CODE REVIEW

**Comment by mal-tee** [comment]
I've used https://stackoverflow.com/a/11849333 to find the ones from the last commit. We could consider adding such a regex to either the CI or a precommit hook. What do you think @baltpeter? :D

File: content/en/verein/transparency.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "type": "page"
 }
 
-<a href="https://www.transparency.de/mitmachen/initiative-transparente-zivilgesellschaft/" class="no-link-decoration"><img src="/img/logo-itz.svg" style="float: right; width: 40%; min-width: 200px; padding: 5px;"></a>
+<a href="https://www.transparency.de/mitmachen/initiative-transparente-zivilgesellschaft/" class="no-link-decoration"><img src="/img/logo-itz.svg" style="float: right; width: 40%; min-width: 200px; padding: 5px;" alt="Initiative Transparente Zivilgesellschaft logo"></a>
```
Comment (baltpeter) [inline content/en/verein/transparency.md:6]:
```suggestion
<a href="https://www.transparency.de/mitmachen/initiative-transparente-zivilgesellschaft/" class="no-link-decoration"><img src="/img/logo-itz.svg" style="float: right; width: 40%; min-width: 200px; padding: 5px;" alt="Logo of the Initiative for a Transparent Civil Society, with the German text “Initiative Transparente Zivilgesellschaft”"></a>
```

**Review by baltpeter — COMMENTED** [review]
Thanks. I agree that most of the card icons to the side are purely decorative and don't need an alt text. Not so sure about the other cases. Some of the humaaans (e.g. the one on the 404 page) are pretty prominent. Also, the icons on the verein page are pretty important imo.

**Comment by baltpeter** [comment]
> I've used [stackoverflow.com/a/11849333](https://stackoverflow.com/a/11849333) to find the ones from the last commit. We could consider adding such a regex to either the CI or a precommit hook. What do you think @baltpeter? :D

I agree that automated checking for a11y issues is a good idea. Not sure about the regex though. I think there are tools that are better suited. Let's start an issue for that.

File: content/de/blog/supervisory-authorities/index.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Dabei ist zu beachten, dass dieser Grundsatz nur regelt, mit welcher Behörde Du
 <a id="finder"></a>
 ## An welche Datenschutz-Aufsichtsbehörde sollte ich mich wenden?
 
-<img class="offset-image offset-image-right" src="/img/humaaans/question-1.svg" style="height: 350px;">
+<img class="offset-image offset-image-right" src="/img/humaaans/question-1.svg" style="height: 350px;" alt="">
```
Comment (mal-tee) [inline content/de/blog/supervisory-authorities/index.md:44]: Should this humaaan get an alt-tag too?

File: content/de/open-source.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 	"heading": "<span style='text-align: center; font-family: monospace;'>Datenanfragen.de <span class='color-red-600' title='liebt'>❤</span> Open Source</span>"
 }
 
-<img id="open-source-humaaan" class="top-left-humaaan" src="/img/humaaans/open-source.svg">
+<img id="open-source-humaaan" class="top-left-humaaan" src="/img/humaaans/open-source.svg" alt="">
```
Comment (mal-tee) [inline content/de/open-source.md:8]: Should this humaaan get an alt-tag too? If yes, please provide a string. :D

**Comment by mal-tee** [comment]
Unfortunately, I can't speak French and Portuguese. What should we do about the corresponding alt tags?

File: src/i18n/de.json [hunk 4]
```diff
@@ -114,6 +114,7 @@
...
         "404-title": "Fehler 404: Seite nicht gefunden",
         "404-body": "Leider existiert die gesuchte Seite auf diesem Server nicht. Vielleicht hast du dich ja einfach in der URL vertippt? Falls der Fehler wieder auftritt, hinterlasse uns doch einen Hinweis auf unserer E-Mail-Adresse <a href='mailto:dev@datenanfragen.de'>dev@datenanfragen.de</a> oder schreib in unseren <a href='https://github.com/datenanfragen/website/issues/new'>Issue-Tracker</a>.",
+        "404-alt": "Eine Person die auf dem 0 eines großen 404-Schildes sitzt. Sie hält ein Buch, in dem eine Seite fehlt.",
```
Comment (baltpeter) [inline src/i18n/de.json:117]:
```suggestion
        "404-alt": "Eine Person die auf der 0 eines großen 404-Schildes sitzt. Sie hält ein Buch, in dem eine Seite fehlt.",
```

File: src/i18n/en.json [hunk 5]
```diff
@@ -116,6 +116,7 @@
         "press-no-releases": "At this time there are no press releases in English.",
         "404-title": "Error 404: Page not found",
         "404-body": "The page you were looking for sadly doesn't exist on this server. Maybe you just have a typo in the URL? If the error persists, perhaps notify us via our email <a href='mailto:dev@datenanfragen.de'>dev@datenanfragen.de</a> or submit an issue in our <a href='https://github.com/datenanfragen/website/issues/new'>issue tracker</a>.",
+        "404-alt": "A person sitting on the 0 of a big 404 sign. They hold a book with a missing page.",
```
Comment (baltpeter) [inline src/i18n/en.json:119]:
```suggestion
        "404-alt": "A person sitting on the 0 of a big 404 sign. They are holding a book with a missing page.",
```

File: layouts/verein/list.html [hunk 6]
```diff
@@ -4,23 +4,23 @@
...
+                    <div class="top-icon" style="font-size: 35px;"><img src="{{ "card-icons/money.svg" | absURL }}" alt="Ein Euro-Zeichen in einem Kreis."></div>
                     <h2>{{ T "verein-home-card-donate-heading" }}</h2>
                     <p>{{ T "verein-home-card-donate-content" }}</p>
                     <a href="{{ ref . "donate" }}" class="button button-primary">{{ T "verein-home-card-donate-button" }}</a>
                 </div>
             </div>
             <div class="col33 ">
                 <div class="verein-home-card">
-                    <div class="top-icon" style="font-size: 38px;"><img src="{{ "card-icons/group.svg" | absURL }}"></div>
+                    <div class="top-icon" style="font-size: 38px;"><img src="{{ "card-icons/group.svg" | absURL }}" alt="Ein Umrisse von drei Portraits."></div>
```
Comment (baltpeter) [inline layouts/verein/list.html:23]:
```suggestion
                    <div class="top-icon" style="font-size: 38px;"><img src="{{ "card-icons/group.svg" | absURL }}" alt="Umrisse von drei Personen."></div>
```

**Review by baltpeter — CHANGES_REQUESTED** [review]

**Comment by baltpeter** [comment]
> Unfortunately, I can't speak French and Portuguese. What should we do about the corresponding alt tags?

It's probably best to create an issue for that.

**Pull request merged**